packages/deploy-config-sub-generator/src/app/index.ts

```diff
@@ -0,0 +1,467 @@
+import { basename, dirname, join } from 'path';
```

Comment:
Can we reduce the size of this and only include lifecycle methods. Abstract the private function out to a new module.

Reply:
There is now only the init, prompting and `_composeWithSubGenerator` methods in this file